src/styles/styles/styles.android.js

```diff
@@ -0,0 +1,38 @@
+import variables from '../variables';
+import baseStyles from './styles.native';
+
+const styles = {
+    ...baseStyles.styles,
+    displayNameText: {
+        marginTop: 4,
+    },
+
+    emojiMessageText: {
+        top: 3,
+        position: 'relative',
+        fontSize: variables.fontSizeEmoji,
+        lineHeight: variables.fontSizeEmojiHeight,
+    },
+
+    inboxEmojiMessageText: {
+        marginTop: 6,
+    },
+
+    inboxMessageText: {
+        marginTop: 1,
+    },
+
+    onlyEmojisText: {
+        marginTop: 3,
+        fontSize: variables.fontSizeOnlyEmojis,
+        lineHeight: variables.fontSizeOnlyEmojisHeight,
+    },
+
+    profileEmojiText: {
+        fontSize: variables.fontSizeEmojiProfile,
+        lineHeight: variables.fontSizeEmojiProfileHeight,
+        top: 4,
+    },
+};
+
+export default {...baseStyles, ...styles};
```

Comment:
Since `baseStyles` is a very large object and shouldn't be imported anywhere else, I'd recommend using `Object.assign` for better performance:

```js

export default Object.assign(baseStyles, styles);
```

Reply:
Yea I used lodash assign method as required by the app standards
```js
 export default _.assign(baseStyles, styles);
```

Reply:
did you forget to push this change?